subsys/bluetooth/host/settings.c

```diff
@@ -119,6 +119,14 @@ static int set(const char *name, size_t len_rd, settings_read_cb read_cb,
 	ssize_t len;
 	const char *next;
 
+	if (!atomic_test_and_set_bit(bt_dev.flags, BT_DEV_ENABLE)) {
```

Comment:
Should be `atomic_test_bit()` not `test_and_set()`

Reply:
Of course! Thank you.